packages/kbn-unified-doc-viewer/src/components/doc_viewer/doc_viewer.tsx

```diff
@@ -22,20 +22,20 @@ export interface DocViewerProps extends DocViewRenderProps {
  * a `render` function.
  */
 export function DocViewer({ docViews, ...renderProps }: DocViewerProps) {
-  const tabs = docViews.map(({ title, render, component }: DocView, idx: number) => {
+  const tabs = docViews.map(({ id, title, render, component }: DocView) => {
     return {
-      id: `kbn_doc_viewer_tab_${idx}`,
+      id: `kbn_doc_viewer_tab_${id}`,
       name: title,
       content: (
         <DocViewerTab
-          id={idx}
+          id={id}
           title={title}
           component={component}
           renderProps={renderProps}
           render={render}
         />
       ),
-      ['data-test-subj']: `docViewerTab-${idx}`,
+      ['data-test-subj']: `docViewerTab-${id}`,
```

Comment:
nit: Not related to the PR.
At line 43 there is a comment, 
> // There there's a minimum of 2 tabs active in Discover.

Can you please remove 1 `there`. Its just triggering my OCD